build-system/babel-plugins/babel-plugin-transform-amp-asserts/test/fixtures/transform-assertions/transform-dev-assert-boolean/input.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+dev().assertBoolean(true);
+let result = dev().assertBoolean(false, 'hello', 'world');
+let result2 = dev().assertBoolean();
```

Comment:
Is this a valid line at all? `assertBoolean` with what value?

Reply:
from a transformation perspective, i should try and handle everything that is valid javascript.

from a closure type system, assertBoolean takes in `*` (any value) to assess if its boolean or not, so in this case the value of the parameter is `undefined` when the parameter is read during runtime